tools/iotools_test.go

```diff
@@ -0,0 +1,76 @@
+package tools_test
+
+import (
+	"bytes"
+	"io"
+	"testing"
+
+	"github.com/github/git-lfs/errors"
+	. "github.com/github/git-lfs/tools"
```

Comment:
A `.` import (called "dot-import") imports all of that packages publicly exported identifiers into the namespace of the current package. This way, I can refer to things that package `tools` exports in the `tools_test` package without having to say `tools.X` (I can instead just refer to `X`).

They are generally discouraged in production code, but I quite like them for tests since it allows me to isolate tests from private fields/methods while still keeping the code clean.